### PR TITLE
[BPK-796] Enable styling tooltip portal

### PR DESCRIPTION
**Pull request by k0nserv**

### CODE REVIEW

**Comment by k0nserv** [comment]
I think we should also change `className` to `portalClassName` while we're at it. This would be more inline with popover

**Comment by k0nserv** [comment]
Rename pushed in f4ee712

File: packages/bpk-component-tooltip/src/BpkTooltipPortal-test.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ describe('BpkTooltipPortal', () => {
       <BpkTooltipPortal
         id="my-tooltip"
         target={<div>target</div>}
-        className="my-custom-class"
+        portalClassName="my-custom-class"
```
Comment (matthewdavidson) [inline packages/bpk-component-tooltip/src/BpkTooltipPortal-test.js]: There should be two tests for this - one for custom `className` and one for `portalClassName`.

This current one is a mixture of the two (from a test name and component prop point of view)

Comment (k0nserv) [inline packages/bpk-component-tooltip/src/BpkTooltipPortal-test.js]: Good point 👍

Comment (matthewdavidson) [inline packages/bpk-component-tooltip/src/BpkTooltipPortal-test.js]: looks good :shipit:

**Comment by k0nserv** [comment]
Hmmm
image

**Pull request merged**